src/main/webapp/app/instructor-course-dashboard/tutor-leaderboard/tutor-leaderboard.component.html

```diff
@@ -4,18 +4,16 @@
             <tr>
                 <th>ID</th>
```

Comment:
please rename it to Nr because it is not a key any more

Reply:
I actually used `#` as we do in other part of the webapp